core/src/main/java/smashdudes/gameplay/state/playerstate/ground/GroundRunningState.java

```diff
@@ -18,14 +23,18 @@ public void innerUpdate(float dt)
         CharacterInputComponent ci = entity.getComponent(CharacterInputComponent.class);
         VelocityComponent v  = entity.getComponent(VelocityComponent.class);
 
-        float speed = v.runSpeed;
+        float speed = 0;
         if(ci.currentState.left)
         {
-            v.velocity.x -= speed;
+            speed -= v.runSpeed;
         }
         if(ci.currentState.right)
         {
-            v.velocity.x += speed;
+            speed += v.runSpeed;
+        }
+        if(Math.abs(speed) > 0)
+        {
+            v.velocity.x = speed;
         }
```

Comment:
Change from accelerating to setting the speed. 
This may change in the future instead use an `acceleration` and `maxSpeed` pair of values.